grype/db/v6/affected_package_store.go

```diff
@@ -100,13 +100,9 @@ type OSSpecifier struct {
 	// MinorVersion is the second field in the VERSION_ID field in /etc/os-release (e.g. 0 in "7.0.1406")
 	MinorVersion string
 
-	// LabelVersion is mutually exclusive to MajorVersion and MinorVersion and tends to represent the
-	// VERSION_ID when it is not a version number (e.g. "edge" or "unstable")
+	// LabelVersion is a string that represents a floating version (e.g. "edge" or "unstable") or is the CODENAME field in /etc/os-release (e.g. "wheezy" for debian 7)
```

Comment:
Oh yea this comment is way clearer. This also makes a lot more sense after our Thursday stream where we talked about this table.